app/lib/features/onboarding/pages/onboarding_space_creation_page.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/features/onboarding/types.dart';
+import 'package:acter/features/onboarding/widgets/create_new_space_widget.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class OnboardingSpaceCreationPage extends ConsumerWidget {
+  final CallNextPage callNextPage;
+
+  const OnboardingSpaceCreationPage({super.key, required this.callNextPage});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(  
+        body: Center(
+          child: Container(
+          constraints: const BoxConstraints(maxWidth: 500),
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                Expanded(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      _buildIcon(context),
+                      const SizedBox(height: 20),
+                      _buildHeadlineText(context),
+                      const SizedBox(height: 10),
+                      _buildDescriptionText(context),
+                    ],
+                  ),
+                ),
+                _buildActionButton(context, L10n.of(context)),
```

Comment:
```suggestion
                _buildActionButton(context),
```

If we are passing `context` then also we required  to pass `L10n.of(context)`?

Reply:
Addressed by this [commit](https://github.com/acterglobal/a3/pull/2930/commits/6cf3884520f9451f52d23e0d0c90b8c747525960)